Sources/App/Websocket/RoomUpdate.swift

```diff
@@ -0,0 +1,15 @@
+import Vapor
+
+struct RoomUpdate: Content {
+    enum UpdateType: Int, Codable {
+        case VoteForCard
+        case SelectCard
+        case VoteForAnswer
+        case GoToGame
+        case GoToVoting
+        case GoToScoring
+    }
+    
+    let updateType: UpdateType
+    let update: String
```

Comment:
What is update of type String?

Reply:
this is the update to the scene, it should be
var roomUpdate: Content

Reply:
TODO: update this code one on one with zach